cohere_vector/challenges/default.json

```diff
@@ -44,16 +44,38 @@
       }
     },
     {
-      "name": "knn-search-10-100",
+      "name": "standalone-search-knn-10-100",
       "operation": "knn-search-10-100",
       "warmup-iterations": 100,
-      "iterations": 1000
+      "clients": {{ standalone_search_clients | default(1) | int }},
+      "iterations": {{ standalone_search_iterations | default(10000) | int }}
     },
     {
-      "name": "knn-search-100-1000",
+      "name": "standalone-search-knn-100-1000",
       "operation": "knn-search-100-1000",
       "warmup-iterations": 100,
-      "iterations": 1000
+      "clients": {{ standalone_search_clients | default(1) | int }},
+      "iterations": {{ standalone_search_iterations | default(10000) | int }}
```

Comment:
Can we also another task to test with 8 clients? Same for knn-10-100.

Reply:
We now have two flavor of the knn searches`*single-client`and `*-multiple-clients` For multiple clients, I kept the number of clients configurable with `8` as a default.